src/core/payment/Model.ts

```diff
@@ -10,7 +10,19 @@ export interface PaymentProcess {
     storePayout: PayoutDetails[],
     maximumHeight: number,
     blocks: Block[],
-    totalPayoutFundsNeeded: number
+    totalPayoutFundsNeeded: number,
+    totals?: PaymentTotals
+}
+
+export interface PaymentTotals {
+    coinBaseSum: number,
+    userCommandTransactionFeeSum: number,
+    freeTransferFromCoinBaseSum: number,
+    netCoinBaseReceveid: number,
+    payoutAmountsSum: number,
+    payoutFeesSum: number,
+    netMinaToPoolOperator: number
+
```

Comment:
We should also summarize transaction fees that will be incurred, and reduce the net to operator by the fees they pay to send the payouts.

Reply:
After reading more - I think that's what payoutFeesSum is. Never mind.